src/6_branch.js

```
@@ -1965,3 +1965,37 @@ Branch.prototype['trackCommerceEvent'] = wrap(callback_params.CALLBACK_ERR, func
 	});
 	done();
 });
+/*
+ logEvent() documentation
+ */
+Branch.prototype['logEvent'] = wrap(callback_params.CALLBACK_ERR, function(done) {
+	var self = this;
+	var name = arguments && utils.validateParameterType(arguments[1], 'string') ? arguments[1] : null;
```

We should probably throw an error if they don't include an event name right?